src/net.h

```diff
@@ -501,7 +505,8 @@ class CNode
         try
         {
             BeginMessage(pszCommand);
-            EndMessage();
+            size_t bytesSent = EndMessage();
+            AccountForSentBytes(pszCommand, bytesSent);
```

Comment:
I like the second proposal more. I modified the PR accordingly, however `EndMessage()` signature needed to be modified as well, since `AccountForSentBytes()` needs the message type.